packages/core/package.json

```diff
@@ -26,12 +26,15 @@
     "body": "^5.1.0",
     "debug": "^2.6.0",
     "http-errors": "^1.6.1",
+    "js-yaml": "^3.9.1",
     "lodash": "^4.17.4",
-    "path-to-regexp": "^1.7.0"
+    "path-to-regexp": "^1.7.0",
+    "swagger2openapi": "^2.6.4"
   },
   "devDependencies": {
     "@loopback/openapi-spec-builder": "^4.0.0-alpha.6",
-    "@loopback/testlab": "^4.0.0-alpha.6"
+    "@loopback/testlab": "^4.0.0-alpha.6",
+    "@types/js-yaml": "^3.9.1"
```

Comment:
Since `js-yaml` is a regular dependency, I think `@types/js-yaml` should be a regular dependency too.

I understand that we should not need these types as long as `js-yaml` is consumed as an internal dependency only. My concern is that if we decide to export it at some point in the future, then we will almost certainly forget to move `@types/js-yaml` to regular `dependencies`, thus creating a bug for users consuming `@loopback/core` from TypeScript.